src/sentry/eventstream/base.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import
+
+from sentry.utils.services import Service
+
+from sentry.tasks.post_process import post_process_group
+
+
+class EventStream(Service):
+    __all__ = (
+        'publish', 'consume',
```

Comment:
I think you were right yesterday when you mentioned only providing one of the methods to start and then filling in the other one later. I'm not exactly sure how the Kafka consumer end of this is going to look yet, so I think it would probably make sense to only define the `publish` method for now?